src/containers/AdminLayout/index.tsx

```diff
@@ -21,6 +21,9 @@ export const AdminLayout: FC<ComponentProps> = ({left, right}) => {
 
   useEffect(() => {
     rightDiv.current?.scrollTo(0, 0);
+    if (window.innerWidth < 768) {
```

Comment:
768px cutoff ensures the nav menu remains open for iPads, but on a Blackberry Playbook for example (600px) it would close:
![image](https://user-images.githubusercontent.com/967811/88126670-12de8400-cba0-11ea-825d-155eac610dd9.png)